Tests/LegacyUnit/Service/EmailServiceTest.php

```diff
@@ -168,8 +168,47 @@ public function sendEmailToAttendeesForEventWithoutRegistrationsNotSendsMail()
     /**
      * @test
      */
-    public function sendEmailToAttendeesUsesFirstOrganizerAsSender()
+    public function sendEmailToAttendeesUsesTypo3DefaultFromAddressAsSender()
```

Comment:
Generally, for all these tests that are about the case where the data is provided in the system, please let's make this case explicit in the test name.